wordpress-persistent-disks/mysql-claim.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: mysql-claim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
```

Comment:
I don't think this field is necessary? If default, please omit. (ditto: below)

Reply:
resolved